packages/material-ui/src/Button/Button.js

```diff
@@ -411,7 +411,15 @@ Button.propTypes = {
    * If `true`, no elevation is used.
    * @default false
    */
-  disableElevation: PropTypes.bool,
+  disableElevation: chainPropTypes(PropTypes.bool, (props) => {
```

Comment:
I don't think that we should touch on the button. I think that it makes sense to set this prop at the theme level while using the outline variant from time to time. The card is difference because raised is meant as a temporary state.